arch/arm64/core/smp.c

```diff
@@ -205,12 +215,15 @@ void arch_sched_ipi(void)
 void mem_cfg_ipi_handler(const void *unused)
 {
 	ARG_UNUSED(unused);
+	unsigned int key = arch_irq_lock();
 
 	/*
 	 * Make sure a domain switch by another CPU is effective on this CPU.
 	 * This is a no-op if the page table is already the right one.
+	 * Lock irq to prevent the interrupt during mem region switch.
 	 */
 	z_arm64_swap_mem_domains(_current);
+	arch_irq_unlock(key);
```

Comment:
This change should go in a different commit.

Reply:
This was my mistake, I accidentally merged from mainline instead of my branch.  I will remove it.

Reply:
Resolved issue.